# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

generate_trajectoryActionGoal action_goal
generate_trajectoryActionResult action_result
generate_trajectoryActionFeedback action_feedback
